schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,562 @@
+$id: 'https://spec.openapis.org/arazzo/1.0/schema/2024-08-01'
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+description: |-
+  The description of OpenAPI Initiative Arazzo v1.0.0 documents
+  without schema validation, as defined by https://spec.openapis.org/arazzo/v1.0.0
+type: object
+properties:
+  arazzo:
```

Comment:
Maybe it makes sense to use enums.
Smth like:
```
{ type: 'string', enum: ['1.0.0'] },
```

Reply:
Th reason I used a pattern is it allows newer versions to be created without a breaking change, particularly for code generation  

This is also consistent with OpenAPI